railways/src/test/java/com/example/railways/common/utilities/DriverManager.java

```diff
@@ -1,59 +1,60 @@
 package com.example.railways.common.utilities;
 
+import com.example.railways.common.constant.Browser;
 import com.example.railways.dataObjects.ConfigFileReader;
 import org.openqa.selenium.Alert;
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
-import java.util.concurrent.TimeUnit;
+import java.time.Duration;
 
 public class DriverManager {
     private static WebDriver driver;
-    private static final ConfigFileReader configFileReader = new ConfigFileReader();
 
     public static WebDriver getDriver() {
         return driver;
     }
 
-    public static void setDriver(String browserType) {
+    public static void setDriver() {
+        Browser browserType = Browser.valueOf(ConfigFileReader.getValue("browser").toUpperCase().replaceAll("\\s+",""));
```

Comment:
Write a specific method instead of use valueOf. valueOf is compare exactly Enum name.